contents/docs/user-guides/experimentation.mdx

```diff
@@ -0,0 +1,67 @@
+-   You can select which users will participate in the Experiment (e.g. paid users, users in a given country, or all users).
+-   Based on your user selection and your goal, we will estimate the number of users exposed and the number of days required for your Experiment to reach significant results.
+-   You can tweak your settings to speed up the experiment or adjust user exposure (e.g. you may not want certain users exposed to your experiment).
+-   The actual experiment and what your user sees is controlled in your own code (see below on [Creating an experiment](#creating-an-experiment)). You ask PostHog which group does this user belong to and then you show any relevant changes in your app or website. Because this is done in your code you can test any number of things, from changing copy in a button, to taking the user to a different page, or showing them a completely different experience.
+-   Once your experiment is running, we'll automatically track your main conversion and let you know which variant leads to the best results. While the experiment is running, we'll also tell you the probability that any given variant is the winning variant.
+-   To ensure your experiment does not introduce any regressions, you can also track secondary metrics. This is particularly useful if you're optimizing a metric that has a natural trade-off with another metric (e.g. you can maximize conversion by reducing your price but then your total revenue may be reduced).
+-   Once your experiment is terminated, you can easily archive and share your results with other team members.
+
+### Significant results
+
+In order for your results and conclusions to be valid, any experiment must have a significant exposure. For instance, if you test a product change and only 1 user sees the change, you can't extrapolate from that single user that the change will be beneficial/detrimental for your entire user base. This is true for any experiment that uses the scientific method (e.g. this is also done when testing new drugs or vaccines).
+
+PostHog computes this significance for you automatically and under the hood (you can read more about how we do this in our Advanced section below). We will let you know if your experiment has reached significant results or not. Once your experiment reaches significant results, it's safe to use those results to reach a conclusion and terminate the experiment.
+
+### Experimentation vs. Feature Flags
+
+Experimentation and Feature Flags are different features because they serve different purposes. **Experimentation** is used to test changes to your product where you want to maximize a specific metric (ex. testing a new flow to maximize conversion rate). **Feature Flags** are used for phased roll-outs (e.g. a new experimental feature you want to make sure works correctly) or as a way to control feature access. More on Feature Flags [here](/docs/user-guides/feature-flags).
```

Comment:
```suggestion
Experimentation and Feature Flags are different features because they serve different purposes. **Experimentation** is used to test changes to your product where you want to maximize a specific metric (eg. testing a new flow to maximize conversion rate). **Feature Flags** are used for phased roll-outs (e.g. a new experimental feature you want to make sure works correctly) or as a way to control feature access. Read our [Feature Flags user guide](/docs/user-guides/feature-flags) for more infomation on this.
```